index.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
+
+function setCase(str, strCase) {
+	return str['to' + strCase + 'Case']();
+}
+
 module.exports = function (str, plural, count) {
 	if (typeof plural === 'number') {
 		count = plural;
-		plural = str + 's';
+
+		var plur = 's';
+		// get the tail of the string to check if the case should be upper or lower
+		var strCase = str.slice(1) === str.slice(1).toUpperCase() ? 'Upper' : 'Lower';
```

Comment:
I'm grabbing the "tail" of the word, like "NICORN" to see if it's all uppercase. This was because I was initially checking if it was lowercase and I didn't want a capitalized word ("Unicorn") to make it evaluate as false. But since I switched to checking for uppercase instead, I can just do `str === str.toUpperCase()` instead here. Does that seem logical?

```js
var strCase = str === str.toUpperCase() ? 'Upper' : 'Lower'
```

We can maybe drop this feature completely, if it's adding too much complexity?